packages/edit-site/src/components/sidebar-edit-mode/page-panels/page-status.js

```diff
@@ -205,20 +215,29 @@ export default function PageStatus( {
 											onChange={ handleTogglePassword }
 										/>
 										{ showPassword && (
-											<TextControl
-												onChange={ ( value ) =>
-													saveStatus( {
-														password: value,
-													} )
-												}
-												value={ password }
-												placeholder={ __(
-													'Use a secure password'
-												) }
-												type="text"
-											/>
+											<div className="edit-site-change-status__password-input">
```

Comment:
We don't use this class name and I don't think we even need a `div` here. We could replace with a fragment(`<></>`), no?

Reply:
@ntsekouras thanks for your feedback, I added the div and class for consistency with the password field in the post editor, see https://github.com/WordPress/gutenberg/blob/90f68c9f5c767d2736940e7db17ccc1de3641708/packages/editor/src/components/post-visibility/index.js#L106
These two forms are very similar and I'd tend to think it would be good to reuse the same code. Even better, consider to abstract in a specific component. I'll create a separate issue if no objections.